src/sentry/models/grouphistory.py

```diff
@@ -24,13 +24,16 @@
 class GroupHistoryStatus:
     # Note that we don't record the initial group creation unresolved here to save on creating a row
     # for every group.
+
+    # UNRESOLVED should be used for ONGOING groups.
+    # Use REGRESSED/ESCALATING to specify other substatuses.
```

Comment:
I'm not too sure about this - we could also deprecate UNRESOLVED and add a new value for ONGOING. Not sure if that deprecation would affect any metrics though